bridge/opentracing/internal/mock.go

```diff
@@ -23,11 +23,10 @@ import (
 
 	otelcorrelation "go.opentelemetry.io/otel/api/correlation"
 	oteltrace "go.opentelemetry.io/otel/api/trace"
+	"go.opentelemetry.io/otel/bridge/opentracing/migration"
 	"go.opentelemetry.io/otel/codes"
 	otelparent "go.opentelemetry.io/otel/internal/trace/parent"
 	"go.opentelemetry.io/otel/label"
-
-	"go.opentelemetry.io/otel/bridge/opentracing/migration"
```

Comment:
Is this a golint change?

Reply:
Yeah, probably worth reverting as it is unrelated and not something the build system is enforcing.

Reply:
Done. Also rebased on latest master.